index.rkt

```diff
@@ -350,7 +350,7 @@ Finally we use @racket[datum->syntax] to convert this back to
 @racket[syntax]:
 
 @i[
-(datum->syntax #f '(values "i" "am" "backwards"))
+(datum->syntax stx '(values "i" "am" "backwards"))
 ]
```

Comment:
You're right this would better remain as `#f`, I'll add a comment on the margin note explaining what `#f` does.